Tests/SwiftDocCTests/Model/DocumentationMarkupTests.swift

```diff
@@ -181,7 +181,108 @@ class DocumentationMarkupTests: XCTestCase {
             XCTAssertEqual(expected, model.abstractSection?.content.map{ $0.detachedFromParent.debugDescription() }.joined(separator: "\n"))
         }
     }
-    
+
+    func testCertainDirectivesAreRemovedFromContent() throws {
+
+        func checkSectionContent(expectedContent: String?, section: Section?) {
+            if let desc = section?.content.map({ $0.detachedFromParent.debugDescription() }).joined(separator: "\n") {
+                XCTAssertEqual(expectedContent, desc, "Found unexpected content: \n\(desc)")
```

Comment:
When adding test helpers like this it's helpful to given them `file` and `line` arguments that default to `#file` and `#line` and then also pass those values to all the `XCTAssert` calls. This makes it so that if the test fails then the failure will be attributed to the location that called `checkSectionContent` instead of the location inside of that function. It barely matters if the test helper is local to the same file, but if it's a general test helper in a separate file it can make a big difference.

```suggestion
        func checkSectionContent(expectedContent: String?, section: Section?, file: StaticString = #file, line: UInt = #line) {
            if let desc = section?.content.map({ $0.detachedFromParent.debugDescription() }).joined(separator: "\n") {
                XCTAssertEqual(expectedContent, desc, "Found unexpected content: \n\(desc)", file: file, line: line)
```